engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -1172,6 +1158,20 @@ namespace dmGameSystem
                 out_value.m_Variant = dmGameObject::PropertyVar(dmPhysics::GetMass2D(component->m_Object2D));
             }
             return dmGameObject::PROPERTY_RESULT_OK;
+        } else if (params.m_PropertyId == PROP_LINEAR_DAMPING) {
```

Comment:
I moved the velocity+mass properties to the top, since those are the more likely ones to set at runtime.